src/Illuminate/Foundation/Console/KeyGenerateCommand.php

```diff
@@ -94,10 +94,18 @@ protected function setKeyInEnvironmentFile($key)
      */
     protected function writeNewEnvironmentFileWith($key)
     {
+        $input = file_get_contents($this->laravel->environmentFilePath());
+
+        $currentKey = $this->laravel['config']['app.key'];
+
+        if (strlen($currentKey) !== 0) {
```

Comment:
You can avoid the overhead of a function call here:

```suggestion
        if ($currentKey !== '') {
```